include/cantera/kinetics/Reaction.h

```diff
@@ -162,6 +162,18 @@ class Reaction
         return bool(m_third_body);
     }
 
+    //! Register a function to be called if setRate is used.
+    //! @param id  A unique ID corresponding to the object affected by the callback.
+    //! @param callback  The callback function to be called after setRate is called.
+    //! When the callback becomes invalid (for example, the corresponding object is
+    //! being deleted), the removeSetRateCallback() method must be invoked.
+    //! @since New in %Cantera 3.1
+    void registerSetRateCallback(void* id, const function<void()>& callback);
+
+    //! Remove the setRate callback function associated with the specified object.
+    //! @since New in %Cantera 3.1
+    void removeSetRateCallback(void* id);
```

Comment:
Since this didn't make it in time for Cantera 3.1, these need to be updated as `New in %Cantera 3.2`. Likewise with the similar new methods in the `Kinetics` class.